addon/components/o-s-s/input-container.ts

```diff
@@ -12,8 +16,15 @@ interface OSSInputContainerArgs {
 }
 
 export default class OSSInputContainer extends Component<OSSInputContainerArgs> {
-  get displayErrorStyle(): boolean {
-    return Boolean(this.args.errorMessage) || Boolean(this.args.hasError);
+  get hasStyleKey(): boolean {
+    return Boolean(this.styleKey);
+  }
+
+  get styleKey(): MessageType | null {
```

Comment:
what does `key` refer to here ? shouldn't it be `type` as implied by the `MessageType` type you created ?